### ISSUE TITLE
AngularJS directive in HTML Element

### ISSUE THREAD

**Issue by D-S91** [formio/ngFormio#683]
This might be a weird question, but I'm trying to show a directive inside a HTML Element.
So i have the formbuilder working on my environment, but when i add a HTML Element with <directive></directive> it doesn't do anything. Should this work or is this not how it was intended?

**Comment by randallknutson** [comment]
This should work assuming you have the right libraries loaded. Both angularjs and ngformio need to be loaded on the page, otherwise the browser won't know how to handle it. Be sure to read all the information in the readme in https://github.com/formio/ngformio

**Comment by D-S91** [comment]
Well i tried it. I have to be honest that I'm not fully into AngularJS. I have the builder working and that works great. But when i place a directive like `<custom-directive></custom-directive>` inside the HTML Element, than it just shows up empty. I think (or guess) it has to do with scoping. 

When i check the DOM at the place it should be showing. All it says is `<div ref="html" class=""></div>` nothing of the mentioned directive shows up...

**Comment by randallknutson** [comment]
If all you are trying to do is get a form to show up, you may want to look at our inline embed script. If you build the form on portal.form.io and go to the Launch tab you can see a single line that can be added to any webpage and it will render the form. That would remove the need for AngularJS.

**Comment by D-S91** [comment]
Hey @randallknutson sorry for the late reply. Week has been hectic.
But i noticed that on my own environment angularjs does run since {{data.firstName}} inside a html element becomes invisible. However when i populate the firstName text field it doesn't work. If i try the demo builder linked on this github, then {{data.firstName}} remains visible. Which to me suggest that the angularJS isn't running.

Basically my question would be which scope does the html element have? is it his own scope or the general scope?

Thanks for your help so far!

**Comment by daneformio** [comment]
Closing this thread as it is outdated. Please re-open if it is still relevant. Thank you for your contribution!
